package.json

```
@@ -499,6 +499,13 @@
             "scope": "machine",
             "type": "string",
             "markdownDescription": "Path to a file (`TF_LOG_PATH`) for Terraform executions to be logged used by the the Terraform Language Server. Support for variables (e.g. timestamp, pid, ppid) via Go template syntax `{{varName}}`"
+          },
+          "terraform.mcp.server.enabled": {
```

```suggestion
          "terraform.mcp.server.enable": {
```

See changelog comment